docs/CONTRIBUTING.md

```diff
@@ -166,6 +166,10 @@ unstable state.
 In order to build a specific component (e.g. `runtime`), please follow the
 following steps.
 
+0. When building the project for the first time or after changing to a new
```

Comment:
Please renumber starting from 1. We want our files to be sensible when not rendered as well!

Reply:
The idea was that this '0' step has to be done only once, but maybe it's only confusing, so I'll change it to start at 1.

Reply:
If you want to convey the "only once" nature, maybe don't make it part of the list? Or put it in a `>` block?

Reply:
I have added a separate section `Bootstrapping build dependencies`: https://github.com/enso-org/enso/pull/928/files#diff-8595297407c24b9ccb87b2a698cca9c2R165-R174